### ISSUE TITLE
Move to a free git hosting platform

### ISSUE THREAD

**Issue by OutOfContainment** [misyltoad/frog-protocols#9]
Community is important.

Linux community is often aligned towards free and open-source initiatives; unfortunately, Github is not one of them. Not only it being a proprietary platform, it is also run by a huge unethical company that has a history of using copyleft-licensed code hosted on Github in their own proprietary products such as Copilot.

There is a lot of reasons why this decision can harm an open-source community and a lot of people who will be kept away from contributing to this project. This is a request to reconsider current hosting platform before it's too late.

There are currently a lot of great alternatives, such as **[Codeberg](https://codeberg.org/)**, **[Gitea](https://about.gitea.com/)** and **[sourcehut](https://sr.ht/)**. 
**[Gitlab](https://gitlab.freedesktop.org/)** might also be a good choice for the sake of close collaboration with original [wayland-protocols](https://gitlab.freedesktop.org/wayland/wayland-protocols).

Thank you for your initiative and I hope it will shed a light on wayland future. 
Let's not turn this place into a froggy pot for all of us 🐸.

**Comment by probonopd** [comment]
> Community is important.

For that reason, open source projects that are meant to attract eyeballs should not be developed in secluded enclaves.

**Comment by OutOfContainment** [comment]
> For that reason, open source projects that are meant to attract eyeballs should not be developed in secluded enclaves.

Precisely, thank you. Git is awesome and it is tragic that current web model made us think that it should be centralized.

Making a codeberg account will take you 5 minutes;
Making github a free platform will take you several years of lost lawsuits and tons of public attention.

**Comment by sounddrill31** [comment]
Though I would prefer it if you guys kept support for github as well

One way I can think of is a kind of sync workflow. Even though I have yet to experiment with those, I've seen similar things floating around

**Comment by probonopd** [comment]
> Precisely, thank you. 

Not sure you understood what I meant to say: People find open source code by searching on GitHub. So being on GitHub is essential. And also the ability to send pull requests via GitHub, because it lowers the barriers to entry significantly.

Yes, I'd prefer an open source solution, too. But then, if no contributors and users are there, it's not very useful. So a carefully planned hybrid approach (mirroring to GitHub, accepting GitHub pull requests and issues) might be the winning strategy. Best of both worlds.

**Comment by OutOfContainment** [comment]
> So being on GitHub is essential.

Yes, indeed, and we are at fault. This is what this issue is about.

Also, if we're being completely honest, I personally would not trust a contribution from someone who does not know what Gitlab (or google search) is. This is extremely subjective and opinionated, but github is more like a social media at this point - you can see the mess at [winamp repository](https://github.com/WinampDesktop/winamp). 

Our subject is a very delicate matter and I would never pick quantity over quality when it comes to protocol standardization.

> hybrid approach

That is a solution I would personally be satisfied with, but it's up to @misyltoad - it will require some work and may create nuances; IMO just switching to codeberg would be much easier.

Best solution would probably be using github as a read-only fork which links to original repository: a lot of projects are following [simular approach](https://github.com/emersion/hello-wayland).

**Comment by p2ndemic** [comment]
If you allow me, as a beginner I’d like to share my opinion. Thanks to GitHub, I’ve found many wonderful open-source projects that I would never have discovered on other isolated platforms!

**SourceHut** is a whole different story - you can’t even create an issue there. At least, I couldn’t figure out how. It seems to me like a place where once-active projects slowly die after being moved there. Personally, I’ve noticed that many of them haven’t been updated for years!

@OutOfContainment  If you want to isolate yourself from the community, please feel free to create private repositories and develop your projects on your own. But please don’t encourage other people to do the same. After all, open source is made for the benefit of people - for us, ordinary users.

I’m not a programmer, but if I were, my mission would be to help ordinary users: to create programs that would make them a little happier and to share them with as many people as possible!
